server/Makefile

```diff
@@ -828,3 +830,34 @@ migrations-extract:
 	@echo Listing migration files
 	@echo "# Autogenerated file to synchronize migrations sequence in the PR workflow, please do not edit.\n#" > channels/db/migrations/migrations.list
 	find channels/db/migrations -maxdepth 2 -mindepth 2 | sort >> channels/db/migrations/migrations.list
+
+test-migration:
+	$(GO) install github.com/mattermost/morph/cmd/morph@latest
+	# apply postgres migrations to the new db
+	bin/morph apply up --driver postgres --dsn "${POSTGRES_DSN}" --path ${POSTGRES_MIGRATIONS_PATH} --number -1
+
+	# create temporary file for the config
+	@cp tests/template.load tests/temp.load
+
+	# set required values for the pgloader config, these values are read from /build/dotenv/migration.env in the CI
+	# do for psql credentials
+	@sed -i'' -e 's|{{ .pg_user }}|${PG_USERNAME}|g' tests/temp.load
+	@sed -i'' -e 's|{{ .pg_password }}|${PG_PASSWORD}|g' tests/temp.load
+	@sed -i'' -e 's|{{ .target_schema }}|${PG_SCHEMA}|g' tests/temp.load
+
+	# do for mysql credentials
+	@sed -i'' -e 's|{{ .mysql_user }}|${MYSQL_USERNAME}|g' tests/temp.load
+	@sed -i'' -e 's|{{ .mysql_password }}|${MYSQL_PASSWORD}|g' tests/temp.load
+	@sed -i'' -e 's|{{ .source_schema }}|${MYSQL_SCHEMA}|g' tests/temp.load
+
+	# install pgloader
+	apt install -y pgloader
```

Comment:
Can we try to install `pgloader` by doing a `docker run` from the workflow file? We typically try to keep the Makefile commands to be platform insensitive.